src/firmware/app/control/trajectory_planner.h

```diff
@@ -0,0 +1,104 @@
+#pragma once
+
+#include "firmware/shared/math/polynomial_2d.h"
+#include "firmware/shared/math/vector_2d.h"
+
+#define TRAJECTORY_PLANNER_MAX_NUM_ELEMENTS                                              \
+    6000  // The maximum size of the array containing trajectory elements
+#define __TRAJECTORY_PLANNER_MAX_PATH_LENGTH__ 18  // [meters]
```

Comment:
Separate question, why is there a max path length?

Reply:
Good question, removed. Was part of some earlier logic